airflow/providers/docker/operators/docker.py

```diff
@@ -318,18 +321,20 @@ def _get_cli(self) -> APIClient:
             tls_config = self.__get_tls_config()
             return APIClient(base_url=self.docker_url, version=self.api_version, tls=tls_config)
 
-    def get_command(self) -> Union[List[str], str]:
+    @staticmethod
+    def get_command(command: Union[str, List[str]]) -> Union[List[str], str]:
         """
         Retrieve command(s). if command string starts with [, it returns the command list)
```

Comment:
Just wondering.. Why do we actually support this? Is it really necessary for users to pass a string containing a list instead of just a list?

Reply:
I suppose it's for supporting jinja templating, for example, `"{{command}}"` where `command` is a list.

Reply:
True. Thanks.